build.gradle

```diff
@@ -87,7 +87,7 @@ publishing {
         	customizePom(pom)
         	groupId project.property('mavenGroupId')
         	artifactId project.property('mavenArtifactId')
-        	version "${mavenMajorVersion}.${mavenMinorVersion}.${mavenPatchVersion}${mavenCentralSnapshotArtifactSuffix}"
+        	version "${mavenMajorVersion}.${mavenMinorVersion}.${mavenPatchVersion}${mavenArtifactSuffix}"
```

Comment:
Can you add a big Todo comment to revert when we GA? And create a corresponding issue so we don't forget about it

Reply:
Done, also reflected in Beta